Kwenta/kwenta
# feat: added Kwenta to spot balances table.

## Description

"**Spot Balance**" update:

- [x] show major token holdings (stables, ETH, OP, etc) including liquid $KWENTA (if balances > 0).
    > only on L2 like exchange page.
- [x] the price of the tokens above is fetched from CoinGecko.
- [x] Rename table header "Oracle Price" to just "Price".
- [x] add $KWENTA token icon.
- [x] add token holdings' USD balance to "Portfolio value" on Dashboard.
- [x] add token holdings' USD balance to "Spot Balance" tab on Dashboard.

## Related issue

Closes: #1630 

## Motivation and Context


## How Has This Been Tested?



- switch to L2 mainnet and go to `/dashboard` to check "Portfolio value" and "Spot Balance" if includes token holdings.

- click "Spot Balance" tab to check if the table below includes major tokens (including $KWENTA if its balance > 0, make sure that its icon is shown too).

- works both on Mobile and PC, and sorting functionality is not broken on the spot balances table.

## Screenshots (if appropriate):

**PC**
https://user-images.githubusercontent.com/31362988/203453115-472b58b4-af1a-4f94-8063-e110f6a7344b.mov
**Mobile**
<img width="401" alt="Screen Shot 2022-11-23 at 09 28 14" src="https://user-images.githubusercontent.com/31362988/203453145-8a432d6f-994d-422c-805a-68c432157855.png">
